tools/devops/automation/templates/build/build.yml

```diff
@@ -594,6 +594,8 @@ steps:
   timeoutInMinutes: 600
   enabled: ${{ parameters.runTests }}
   env:
+    ${{ if eq(variables['Build.SourceBranchName'], 'main') }}: # only works if you have a main branch
```

Comment:
This means that we won't run the .NET tests after branching for .NET releases. Is this what we want?